packages/pv-scripts/scripts/dev.js

```diff
@@ -24,38 +21,39 @@ const { prepareWebpackConfig } = require("../helpers/prepareWebpackConfig");
 const { getCompiler } = require("../helpers/devServerHelpers");
 const isInteractive = process.stdout.isTTY && autoConsoleClear();
 
-prepareWebpackConfig("development")
-  .then(webpackConfig => {
-    // Create a webpack compiler that is configured with custom messages.
-    const compiler = getCompiler({
-      webpackConfig,
-      webpack,
-    });
-
-
-    const devServerConfig = webpackConfig[0].devServer;
+prepareWebpackConfig("development").then(webpackConfig => {
+  // Create a webpack compiler that is configured with custom messages.
+  const compiler = getCompiler({
+    webpackConfig,
+    webpack
+  });
 
-    const devServer = new WebpackDevServer(compiler, devServerConfig);
+  const devServerConfig = webpackConfig[0].devServer;
 
-    const devServerUrl = `http${devServerConfig.https ? "s" : ""}://${devServerConfig.host}:${devServerConfig.port}`;
+  const devServer = new WebpackDevServer(compiler, devServerConfig);
 
+  const devServerUrl = `http${devServerConfig.https ? "s" : ""}://${
```

Comment:
maybe we should disable prettier for this line.

Reply:
added prettier-ignore